packages/pages-shared/asset-server/handler.ts

```diff
@@ -26,6 +26,9 @@ type BodyEncoding = "manual" | "automatic";
 // and if so, serve it from the cache instead of responding with a 404.
 // This gives a bit of a grace period between deployments for any clients browsing the old deployment.
 export const ASSET_PRESERVATION_CACHE = "assetPreservationCache";
```

Comment:
How about renaming this to `ASSET_PRESERVATION_CACHE_V1` to make it really clear what is going on below?

Reply:
Good idea, done!